### ISSUE TITLE
An error occurred when the script was running

### ISSUE THREAD

**Issue by lpy-199969** [TOR968/ClaytonBot#5]
Error processing account keke3: Cannot read properties of undefined (reading 'dailyReward')

**Comment by Lastnet0709** [comment]
i just try with new Query and it's still working

**Comment by xndjzx** [comment]
Yeah new query. Me too. But there are an error even the script run. Maybe the task or somethings.

**Comment by TOR968** [comment]
Action plan if you have an error:
- check the availability of updates;
- restart the code;
- check if the game is available and if errors in the console when trying to do something;
- update urlid;
- update queryid/userid;
- if none of the above helped, go to [tg channel](https://t.me/tor_dev) there write about the error, if possible I will answer).
